presto-docs/src/main/sphinx/functions/comparison.rst

```diff
@@ -120,3 +120,34 @@ The following types are supported:
 .. function:: least(value1, value2) -> [same as input]
 
     Returns the smallest of the provided values.
+
+Quantified Comparison Predicates: ALL, ANY and SOME
+---------------------------------------------------
+
+``ALL``, ``ANY`` and ``SOME`` quantifiers can be used together with comparison operators in the
+following way::
+
+    <expression> <comparison operator> <quantifier> (<subquery>)
+
+For example::
+
+    SELECT 'hello' = ANY (VALUES 'hello', 'world'); -- true
+
+    SELECT 21 < ALL (VALUES 19, 20, 21); -- false
+
+    SELECT 42 >= SOME (SELECT 41 UNION ALL SELECT 42 UNION ALL SELECT 43); -- true
+
+Here are the meanings of some quantifier and comparison operator combinations:
+
+====================    ===========
+Expression              Meaning
+====================    ===========
+``A = ALL (...)``       Evaluates to ``true`` when A is equal to all values returned by a subquery.
```

Comment:
Remove the " returned by a subquery" from all these. It makes it easier to read.